pkg/platform/kube/test/suite.go

```diff
@@ -265,7 +265,8 @@ func (suite *KubeTestSuite) WaitForFunctionState(getFunctionOptions *platform.Ge
 
 func (suite *KubeTestSuite) deployAPIGateway(createAPIGatewayOptions *platform.CreateAPIGatewayOptions,
 	onAfterIngressCreated OnAfterIngressCreated,
-	expectError bool) {
+	expectError bool,
+	errorRootCauseMessage string) {
```

Comment:
a small change with big impact, assuming tomorrow you'd want to resolve the err instance itself, it would even become more complicated. see my comment below